DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: worldfootballR
 Title: Extract and Clean World Football (Soccer) Data
-Version: 0.6.2.7000
+Version: 0.6.2.7100
```

Comment:
this is more or less a patch, so i'm not sure a full version update is justified

Reply:
Agree